packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -2799,6 +2799,82 @@ void main() {
       expect(radioValue, 1);
     });
   });
+
+  testWidgets("MenuItemButton doesn't close when pressed", (WidgetTester tester) async {
```

Comment:
nit: maybe change the message to something like "MenuItemButton respects the closeOnActivate property" because by default, a MenuItemButton can close menu when it is pressed:)